static/webpack.config.js

```diff
@@ -23,6 +23,7 @@ const config = {
     dev: [__dirname + "/js/dev.ts", __dirname + "/css/dev.scss"],
     dev_menu: __dirname + "/js/dev_menu.ts",
     download: __dirname + "/js/download.js",
+    dev_menu: [__dirname + "/js/dev_menu.ts", __dirname + "/css/timeline.scss"],
```

Comment:
please make a new scss file, pulling in the components you need.

Reply:
Added "/css/timeline_statsvar.scss"